lifecycle/lifecycle-runtime-compose/build.gradle

```diff
@@ -42,6 +42,9 @@ androidXComposeMultiplatform {
     darwin()
     js()
     wasm()
+
+    linuxX64()
+    linuxArm64()
 }
```

Comment:
This block now matches https://github.com/JetBrains/compose-multiplatform-core/blob/f9b90c58dfb166be398917f1e43326f6566e50ff/compose/runtime/runtime/build.gradle#L66-L75

It would be really nice to abstract this out so we're not always playing whack-a-mole on trying to keep everything in sync.